src/Lesson1/HomeWorkApp.java

```diff
@@ -0,0 +1,51 @@
+package Lesson1;
+
+public class HomeWorkApp {
+
+    public static void main(String[] args) {
+
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+    }
+
+    public static void printThreeWords() {
+        System.out.println("Orange");
+        System.out.println("Banana");
+        System.out.println("Apple");
+    }
+
+    public static void checkSumSign() {
+        int a = 1;
+        int b = -5;
+
+        if ((a + b) >= 0) {
+            System.out.println("Сумма положительная");
+        } else {
+            System.out.println("Сумма отрицательная");
+        }
+    }
+
+    public static void printColor() {
+        int value = 101;
+
+        if (value <= 0)
+            System.out.println("Красный");
+        else if (value > 0 && value <= 100)
```

Comment:
```suggestion
        else if (value <= 100)
```
поскольку данное условие работает само по себе из-за неактульности первого `value<= 0 -> false <=> value > 0`